salt/elasticfleet/tools/sbin_jinja/so-elastic-fleet-optional-integrations-load

```diff
@@ -155,6 +156,9 @@ if [[ -f $STATE_FILE_SUCCESS  ]]; then
         # Write out file for generating index/component/ilm templates
         latest_installed_package_list=$(elastic_fleet_installed_packages)
         echo $latest_installed_package_list | jq '[.items[] | {name: .name, es_index_patterns: .dataStreams}]' > $PACKAGE_COMPONENTS
+        # Refresh installed component template list
+        latest_component_templates_list=$(so-elasticsearch-query _component_template | jq '.component_templates[] | .name' | jq -s '.')
+        echo $latest_component_templates_list > $COMPONENT_TEMPLATES
```

Comment:
What if this request fails with an error message and is then echo'ed - what kind of problems will that cause?